### ISSUE TITLE
change cubicConversionError default to match cu2qu

### ISSUE THREAD

**Issue by anthrotype** [googlei18n/ufo2ft#141]
In light of the changes in cu2qu v1.2.0 https://github.com/googlei18n/cu2qu/releases/tag/v1.2.0

**Comment by anthrotype** [comment]
(sorry I pressed the "Comment" button too early 😁 )

So, the problem is that currently ufo2ft uses a different `cubicConversionError` default value from the default value which cu2qu and (following the latter) fontmake is using.

In ufo2ft, this is defined in absolute font units, and the value is currently `2`, e.g. it is 2 units whether the UPEM is 1000 or 2048.

In cu2qu, the default is defined as a factor of the EM square; it was `0.0025`, and it has been changed to `0.001` since v1.2.0.

I propose we change the ufo2ft default so that it matches cu2qu's one. That means changing the default value of `cubicConversionError` keyword argument to be `None`, meaning use the cu2qu's default value (like we do in fontmake). And also, if a non-None value is specified, then that should be interpreted as a factor of the EM square (we can get the UPEM from the defcon font instance).

This would be an API breaking change, but I suspect not many people are using that `cubicConversionError` argument.

I'm cc-ing here the people I remember having seen using ufo2ft compile function directly instead of via fontmake: @adrientetar @khaledhosny
